proxies/live/apiproxy/policies/AssignMessage.HandleRequestIDs.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.HandleRequestIDs">
+    <Set>
+        <Headers>
+            <Header name="NHSD-Request-ID">{message.header.X-Request-ID}</Header>
+            <Header name="NHSD-Correlation-ID">{message.header.X-Request-ID}.{message.header.X-Correlation-ID}.{messageid}</Header>
```

Comment:
What if they haven't set X-Request-ID or X-Correlation-ID? You'll get `..<messageid>` - is that desired?

Reply:
I think so - to make obvious which part is which ID